Avalonia.sln

```diff
@@ -39,6 +39,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{A689DE
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
 		src\Shared\ModuleInitializer.cs = src\Shared\ModuleInitializer.cs
+		src\Shared\RawEventGrouping.cs = src\Shared\RawEventGrouping.cs
```

Comment:
Was this file omitted before by mistake? Since it seems to be unrelated to your PR.

Reply:
I've removed it from sln.